lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/migration/BackupManifest.json

```diff
@@ -0,0 +1,15 @@
+{
+	"description": "Manifest of all metadata associated with a single backup file",
+	"properties": {
+		"originalRequest": {
+			"$ref": "org.sagebionetworks.repo.model.migration.BackupTypeRangeRequest"
+		},
+		"secondaryTypes": {
```

Comment:
We might need an additional field containing the logical backup id for secondaries. This might be necessary because if we turn a secondary to a primary the new object will most likely have a different backup id but the restore should still use the old backup id.